trieste/acquisition/rule.py

```diff
@@ -368,7 +381,8 @@ def _remove_point(pending_points: TensorType, point_to_remove: TensorType) -> Te
 
         tf.debugging.assert_shapes(
             [(self.pending_points, [None, "D"]), (points_to_remove, [None, "D"])],
-            message=f"""Point to remove shall be 1xD where D is the last dimension of pending points.
+            message=f"""Point to remove shall be 1xD where D is the last dimension
```

Comment:
Probably a bit cleaner like this (though really I should go through the codebase an clean all these up so they indent properly when raised)

```suggestion
            message=f"""
            Point to remove shall be 1xD where D is the last dimension of pending points.
            Got {tf.shape(self.pending_points)} for pending points
            and {tf.shape(points_to_remove)} for other points.
            """,
```